docs/tasks_snapclones.md

```diff
@@ -6,8 +6,72 @@ sidebar_label: Snapshots and Clones
 ------
 
 ## Snapshots using Kubectl
+For taking a snapshot, you must have a PersistentVolumeClaim for which you need to take snapshot. Once you delpoy the application , it's 
+PVC will be created by the name specified in the application deployment yaml.
+Once the application is running, you take a snapshot. To take a snapshot you must reference the PersistentVolumeClaim name in the snapshot specification that references the data for which you want to take snapshot. An example of `snapshot.yaml` file given below.
```

Comment:
... ,you can take a snapshot.
you must reference = you must refer
that references = that refers
file is given below.